lesson_07/script.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+// ローディング作る
+const addLoading = () => {
+  const loading = document.getElementById("js-loading");
+  const gif = document.createElement("img");
+  gif.src = "loading-circle.gif";
+  loading.appendChild(gif);
+};
+
+// ローディング消す
+const removeLoading = () => document.getElementById("js-loading").remove();
+
+const ul = document.getElementById("js-list");
```

Comment:
Variables here, but I don't think they need to be global variables now.
So
getData.then((value) => {
  const ul = document.getElementById("js-list");
...
});
I think it is fine to use the following.